x-pack/platform/plugins/shared/dataset_quality/public/components/quality_indicator/index.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { dynamic } from '@kbn/shared-ux-utility';
+import type { CoreStart } from '@kbn/core-lifecycle-browser';
+import { PerformanceContextProvider } from '@kbn/ebt-tools';
+import type { DatasetQualityDetailsController } from '../../controller/dataset_quality_details';
+import type { DatasetQualityStartDeps } from '../../types';
+import type { ITelemetryClient } from '../../services/telemetry';
+import { useKibanaContextForPluginProvider } from '../../utils';
+import type { DatasetQualityDetailsContextValue } from '../dataset_quality_details/context';
+import { DatasetQualityDetailsContext } from '../dataset_quality_details/context';
+
+export * from './percentage_indicator';
+export * from './dataset_quality_indicator';
+
+const OverallDataQualityIndicator = dynamic(() => import('./overall_data_quality_indicator'));
+
+interface DatasetQualityIndicatorProps {
+  controller: DatasetQualityDetailsController;
```

Comment:
I don't think this is the way to go, If we want to actually expose this one as a standalone component I think we should have a separate controller, this would mean create a separate (fairly simple) machine that will be in charge on getting just thee information needed for this component to be rendered. I could use underneath the same services, endpoints, etc, but sharing the `datasetqualityDetails` state to this one imo is an overkill